### PR TITLE
docs: update local kubenetes configuration README.md

### PR DESCRIPTION
**Pull request by roaminggypsy**

### CODE REVIEW

**Comment by wfa-reviewable** [comment]
This change is [Reviewable](https://reviewable.io/reviews/world-federation-of-advertisers/cross-media-measurement/1928)

**Review by SanjayVas — APPROVED** [review]
> Reviewed 1 of 1 files at r1, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/world-federation-of-advertisers/cross-media-measurement/1928)* status: all files reviewed, 1 unresolved discussion (waiting on @roaminggypsy)

___
*[`src/main/k8s/local/README.md` line 235 at r1](https://reviewable.io/reviews/world-federation-of-advertisers/cross-media-measurement/1928#-OC5brNjBmKpir2-ZiRz:-OC5brNjBmKpir2-ZiS-:b-2npek4) ([raw file](https://github.com/world-federation-of-advertisers/cross-media-measurement/blob/9f0f224bb1871fc5b591459016773bfced2df3bc/src/main/k8s/local/README.md#L235)):*
> ```Markdown
> 
> ```shell
> kubectl apply -f https://github.com/cert-manager/cert-manager/releases/download/v1.13.2/cert-manager.yaml
> ```

nit: also update cert manager version.

Basically, this file needs to be kept in sync with `.github/actions/install-otel-operator/action.yml`



_Suggestion:_
```Markdown
v1.14.5
```

**Comment by roaminggypsy** [comment]
I added some more changes covering some common operations.

**Review by SanjayVas — CHANGES_REQUESTED** [review]
> Reviewed all commit messages.
> *[Reviewable](https://reviewable.io/reviews/world-federation-of-advertisers/cross-media-measurement/1928)* status: 0 of 1 files reviewed, 4 unresolved discussions (waiting on @roaminggypsy)

___
*[`src/main/k8s/local/README.md` line 55 at r2](https://reviewable.io/reviews/world-federation-of-advertisers/cross-media-measurement/1928#-OCADSql8sV3rJBviFwm:-OCADSql8sV3rJBviFwn:bxiaytr) ([raw file](https://github.com/world-federation-of-advertisers/cross-media-measurement/blob/a42a6b9fae98f9d8552244f4bd68f685309da516/src/main/k8s/local/README.md#L55)):*
> ```Markdown
> ```
> 
> ## Delete Existing Local Cluster
> ```

This should be moved down to the appendix as it only applies to Kind (as opposed to Minikube or K3s)
___
*[`src/main/k8s/local/README.md` line 255 at r2](https://reviewable.io/reviews/world-federation-of-advertisers/cross-media-measurement/1928#-OCADi6R4nUxXLcBiFK-:-OCADi6SCIintKDIsU6S:bfqf7x3) ([raw file](https://github.com/world-federation-of-advertisers/cross-media-measurement/blob/a42a6b9fae98f9d8552244f4bd68f685309da516/src/main/k8s/local/README.md#L255)):*
> ```Markdown
> This depends on [cert-manager](https://github.com/cert-manager/cert-manager), so
> we install that first. Make sure their versions are the same as versions listed in
> [install-otel-operator/action.yml](https://github.com/world-federation-of-advertisers/cross-media-measurement/blob/6f9b14b912099d0ce0ddd02f67b67377c7a3daa9/.github/actions/install-otel-operator/action.yml#L4):
> ```

Use a relative link for anything that's in the same repo



_Code quote:_
```Markdown
(https://github.com/world-federation-of-advertisers/cross-media-measurement/blob/6f9b14b912099d0ce0ddd02f67b67377c7a3daa9/.github/actions/install-otel-operator/action.yml#L4
```
___
*[`src/main/k8s/local/README.md` line 344 at r2](https://reviewable.io/reviews/world-federation-of-advertisers/cross-media-measurement/1928#-OCAEN7t1hadKc6qwznS:-OCAEN7t1hadKc6qwznT:bmk89nv) ([raw file](https://github.com/world-federation-of-advertisers/cross-media-measurement/blob/a42a6b9fae98f9d8552244f4bd68f685309da516/src/main/k8s/local/README.md#L344)):*
> ```Markdown
> ## Debugging Tips
> 
> List cron jobs:
> ```

nit: Leave out all the common kubectl commands, or mention them very generally. These are basics, and there are more forms than are mentioned here. For example, we frequently debug Deployment objects so you're more likely to need `kubectl get deployments`. We also are more likely to want to stream logs for a running Pod or Deployment, where we'd use `kubectl logs -f deployments/foo`
___
*[`src/main/k8s/local/README.md` line 373 at r2](https://reviewable.io/reviews/world-federation-of-advertisers/cross-media-measurement/1928#-OCAF0pw-PVgVYqR1emv:-OCAF0pw-PVgVYqR1emw:bona4xj) ([raw file](https://github.com/world-federation-of-advertisers/cross-media-measurement/blob/a42a6b9fae98f9d8552244f4bd68f685309da516/src/main/k8s/local/README.md#L373)):*
> ```Markdown
> 
> an example: kubectl edit cronjob/measurement-system-prober-cronjob 
> ```shell
> ```

This is not a shell script, but rather a snippet from a YAML file. The above command is though. You also want to leave a blank line before/after code fences.



_Suggestion:_
```Markdown

```yaml
```

**Review by roaminggypsy — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/world-federation-of-advertisers/cross-media-measurement/1928)* status: 0 of 1 files reviewed, 3 unresolved discussions (waiting on @SanjayVas)

___
*[`src/main/k8s/local/README.md` line 55 at r2](https://reviewable.io/reviews/world-federation-of-advertisers/cross-media-measurement/1928#-OCADSql8sV3rJBviFwm:-OCAIQ7-6KJLx-bRdo51:b-896fix) ([raw file](https://github.com/world-federation-of-advertisers/cross-media-measurement/blob/a42a6b9fae98f9d8552244f4bd68f685309da516/src/main/k8s/local/README.md#L55)):*
<details><summary><i>Previously, SanjayVas (Sanjay Vasandani) wrote…</i></summary><blockquote>

This should be moved down to the appendix as it only applies to Kind (as opposed to Minikube or K3s)
</blockquote></details>

Done.
___
*[`src/main/k8s/local/README.md` line 255 at r2](https://reviewable.io/reviews/world-federation-of-advertisers/cross-media-measurement/1928#-OCADi6R4nUxXLcBiFK-:-OCAJ2Mq1nrQwpBQzQiM:b-896fix) ([raw file](https://github.com/world-federation-of-advertisers/cross-media-measurement/blob/a42a6b9fae98f9d8552244f4bd68f685309da516/src/main/k8s/local/README.md#L255)):*
<details><summary><i>Previously, SanjayVas (Sanjay Vasandani) wrote…</i></summary><blockquote>

Use a relative link for anything that's in the same repo
</blockquote></details>

Done.
___
*[`src/main/k8s/local/README.md` line 373 at r2](https://reviewable.io/reviews/world-federation-of-advertisers/cross-media-measurement/1928#-OCAF0pw-PVgVYqR1emv:-OCAJKY66xjXJpG2JKF-:b-896fix) ([raw file](https://github.com/world-federation-of-advertisers/cross-media-measurement/blob/a42a6b9fae98f9d8552244f4bd68f685309da516/src/main/k8s/local/README.md#L373)):*
<details><summary><i>Previously, SanjayVas (Sanjay Vasandani) wrote…</i></summary><blockquote>

This is not a shell script, but rather a snippet from a YAML file. The above command is though. You also want to leave a blank line before/after code fences.
</blockquote></details>

Done.

**Review by stevenwarejones — APPROVED** [review]
> Reviewed 1 of 1 files at r3, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/world-federation-of-advertisers/cross-media-measurement/1928)* status: <span>:shipit:</span> complete! all files reviewed, all discussions resolved (waiting on @roaminggypsy)

**Review by roaminggypsy — COMMENTED** [review]
:lgtm:

> *[Reviewable](https://reviewable.io/reviews/world-federation-of-advertisers/cross-media-measurement/1928#-:-OCePJwG24EWjawbp_GA:bnfp4nl)* status: 0 of 1 files reviewed, all discussions resolved (waiting on @SanjayVas and @stevenwarejones)
